LibertyDSNP/example-client
# Feature/login with wallet 177347812

# Problem
- As a user, I obtain a social identity in the form of a smart contract by creating  an account or logging in with Torus.
- As a user, I am able to create multiple social identities.

[link to Pivotal Tracker #177515975](https://www.pivotaltracker.com/story/show/177515975)

# Solution
- Look to Torus integration in friendly
- Basically copy over what we need

with @claireolmstead 

## Change summary:
- create fake sdk file
- add torus config files/setup
- add `onAuthenticate()` and `logout()` functionality
- add necessary `/services` and `/utilities` files
- rearrange file structure for scss and test files into their own directories
- add styling to login/logout button
- add `antd-overrides.scss`
- add `.stylelintignore` file
- get tests working

## Steps to Verify:

1. Make sure it works for the following cases:
- case 1: no Torus connected
- connect the wallet

case 2: I do not have a dsnp identity
- create a dsnp identity
- sdk call

case 3: I already have a dsnp identity
- I can just login with that identity
- sdk call

## Screenshots (optional):
<img width="843" alt="Screen Shot 2021-03-26 at 4 09 10 PM" src="https://user-images.githubusercontent.com/43625033/112701419-d6a9a480-8e4d-11eb-9c74-0692d5bd95b6.png">
<img width="852" alt="Screen Shot 2021-03-26 at 4 09 29 PM" src="https://user-images.githubusercontent.com/43625033/112701426-d90bfe80-8e4d-11eb-89f6-e3923e62766c.png">
